server/gameconnection.py

```diff
@@ -303,23 +343,26 @@ def send_message(self, message):
             :param teamkiller_id: teamkiller id
             :param teamkiller_name: teamkiller nickname - Used as a failsafe in case ID is wrong
         """
-                
+
         async with self._db.acquire() as conn:
             """
                 Sometime the game sends a wrong ID - but a correct player name
                 We need to make sure the player ID is correct before pursuing
             """
 
-            check = await conn.execute(select([login.c.id]).where(
-                login.c.id == teamkiller_id or
-                login.c.login == teamkiller_name
-            ))
+            check = await conn.execute(
+                select([login.c.id]).where(
+                    login.c.id == teamkiller_id
+                    or login.c.login == teamkiller_name
```

Comment:
this is very invalid query., please, if you are touching it, fix it. or condition in sqlalchemy is `or_`

Reply:
see more https://stackoverflow.com/questions/7942547/using-or-in-sqlalchemy

Reply:
Good catch we should definitely fix that